src/client/lobby/room-list.js

```diff
@@ -0,0 +1,11 @@
+import React, { Component } from 'react';
+
+import config from '../config/config';
+
+export default function RoomList(props) {
+    return (
+        <div className='col-lg-4'>
```

Comment:
We are going for single quotes in js, but when you're writing standard html properties usually have double quotes. Other mentors - what are your thoughts on this when using simple strings in JSX?